WalletWasabi/Crypto/StrobeProtocol/StrobeHasher.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using NBitcoin;
+
+namespace WalletWasabi.Crypto.StrobeProtocol
+{
+	public sealed class StrobeHasher
+	{
+		private Strobe128 strobe;
+
+		private StrobeHasher()
+		{
+			strobe = new Strobe128("WabiSabi_v1.0");
+			Append("domain-separator", "round-parameters");
```

Comment:
these should be made into protocol constants, there is some redundancy with the zk proof `Transcript` class.

Reply:
Done